### DISCUSSION TITLE
Feature Idea: New Episodes Playlist

### DISCUSSION THREAD

**Discussion by backpack5689** [Sn8z/Poddr#1061] [Ideas]
Hey!

So far, I have been *loving* this app! I use it at work on my Windows machine, and then when I get home on my arch laptop. It works and looks beautifully!

One thing that I wish was a thing though would be a third tab in the "Favorites" section, for the New episodes. My idea is that it would take all the favorites currently in your favorites, and then display them in order of date released in the tab. 

![image](https://user-images.githubusercontent.com/47483901/136975995-cf43e858-025a-4c37-9761-880183abb069.png)

Of course, if there's any questions on this request, I'd love to give more input, or I could learn TypeScript and look into contributing :)

**Comment by Sn8z** [comment]
Heya @backpack5689!
I'm sry for taking this long to respond to this, for some reason I'm not getting notifications about new discussions so I've completely missed this...
Thank you for the kind words, I'm happy that you find Poddr useful 😃 
This is great feedback and as a matter of fact there's a similar feature implemented in the development branch!
There's a new third tab where the newest episodes from your favourites are being shown in a separate feed.

Currently this is causing some performance overhead that I'm looking in to get fixed before a release.

I'm also planning more advanced filtering and sorting functions for your favourites so I'll keep your feedback in mind 🕺

**Reply by Sn8z** [reply]
![latest](https://user-images.githubusercontent.com/10712909/154799413-dc68cabe-6d04-41af-9af8-adbc9f22d306.png)
Here's a quick preview of the current development branch, it's still a bit rough around the edges but we are getting there 😄 
And again @backpack5689, sorry for the delayed response!

**Reply by backpack5689** [reply]
No worries! It's not a problem at all! Totally get being caught up and stuff! Excited to see that feature release 😁

**Comment by carltm** [comment]
Any updates?  I just discovered poddr and really like it.  I'd love to see some feature enhancements!

**Reply by Sn8z** [reply]
Sorry for the slow progress atm, things are moving but I'm currently trying different stuff to see in which way I want to take Poddr in the future :)
Currently experimenting with stuff like cloud sync, Flutter as an alternative framework and different podcast APIs to work against. These things take time and might not result in anything that is visible in the repo 🕺 
Also thanks for the generous donation! Means a lot 👍
